data/mods/Aftershock/mobs/robots.json

```diff
@@ -626,8 +564,8 @@
     "id": "mon_milbot_40mm",
     "type": "MONSTER",
     "copy-from": "mon_milbot_base",
-    "name": "grenadier robot",
-    "description": "A military robot still operating due to its internal power core.  This one is armed with an electric prod and an integrated 40mm grenade launcher.",
+    "name": "Wraitheon Suppressor",
+    "description": "Looking more like a tank than a robot, the Suuppressor is built to deal with vehicles, tanks, and personnel alike.  Equipped with a 40mm grenade launcher, it can easily dispatch anything in its path.",
```

Comment:
```suggestion
    "description": "Looking more like a tank than a robot, the Suppressor is built to deal with vehicles (including tanks) and personnel alike.  Equipped with a 40mm grenade launcher, it can easily dispatch anything in its path.",
```